Daf.Meta.Editor/ViewModels/StagingViewModel.cs

```diff
@@ -124,10 +124,14 @@ private void DeleteStagingColumn()
 			if (SelectedDataSource == null || SelectedColumn == null)
 				throw new InvalidOperationException();
 
-			SelectedDataSource.RemoveStagingColumn(SelectedColumn.StagingColumn);
+			SelectedDataSource.DataSource.RemoveStagingColumn(SelectedColumn.StagingColumn);
 
 			// Remove the view model column from the list.
 			StagingColumns.Remove(SelectedColumn);
+
+			// Inform subscribers that the StagingColumn is being removed.
+			// This doesn't get called until I make the corresponding change in Model.
```

Comment:
What does this comment mean?

Reply:
Reminder for myself, not needed anymore. Removed.